github/provider.go

```diff
@@ -10,39 +10,29 @@ func Provider() terraform.ResourceProvider {
 		Schema: map[string]*schema.Schema{
 			"token": {
 				Type:        schema.TypeString,
-				Optional:    true,
+				Required:    true,
```

Comment:
Please make it optional to support operations suggested by GitHub itself: point 2 of https://developer.github.com/v3/guides/best-practices-for-integrators/ is to whitelist GitHub IPs on for WebHook calls coming into the client infrastructure. That call ( https://api.github.com/meta ) does not require a token. Making the token required in the provider schema will make Terraform users' code unnecessarily complicated in order to provide a bit of information which isn't then used.

Reply:
This is good news. Will look to incorporate this into a follow up commit.